tests/reporter_test.py

```diff
@@ -29,20 +29,33 @@ def setup_method(self, method):
 
         self.metrics_header = ["Metric", "Task", "Baseline", "Contender", "Diff", "Unit", "Diff %"]
         self.metrics_data = [
-            ["Min Throughput", "index", "17300", "18000", "700", "ops/s", "4.04%"],
-            ["Median Throughput", "index", "17500", "18500", "1000", "ops/s", "5.71%"],
-            ["Max Throughput", "index", "17700", "19000", "1300", "ops/s", "7.34%"],
+            ["Min Throughput", "index", 17300, 18000, 700, "ops/s", "4.04%"],
+            ["Median Throughput", "index", 17500, 18500, 1000, "ops/s", "5.71%"],
+            ["Max Throughput", "index", 17700, 19000, 1300, "ops/s", "7.34%"],
+            ["Bubblegum", "", 1.7, 1.9, 0.2, "bubbles/flavonoid", "11.71"],
         ]
-        self.numbers_align = "right"
 
-    def test_formats_as_markdown(self):
+    def test_formats_as_markdown_empty(self):
+        self.numbers_align = "ignored"
```

Comment:
Can you please avoid using a class member here? Either by fully inlining it in the function call or just calling it `numbers_align` without self. That will be one less thing to worry about if we want to run tests in parallel one day. :)

Reply:
Sure! I'm all for contained tests. For what it's worth in junit-land it spins up entirely new instances of the test class for each run. I have no idea what pytest does though. *and* junit doesn't run tests for the same class in parallel. At least ours doesn't. Either way. Sure!